### DISCUSSION TITLE
[ponzi Alert] fsbcompany

### DISCUSSION THREAD

**Discussion by micro73** [Webamoozcom/warning-list#146] [Investigation بررسی]
پروژه fsb company
ادرس سایت : fsbcompany.com
توضیحات سایت :

![6](https://user-images.githubusercontent.com/103103225/183713206-7d727ad3-d487-401a-98b3-96adad10e7cf.jpg)

ترجمه ماشینی:
برنامه نویسی، و به خصوص طراحی اپلیکیشن موبایل، یک کسب و کار پرسود است و بسیاری به این صنعت روی آورده اند. اما صرف ساختن و طراحی یک اپلیکیشن ممکن است درآمدی را که انتظارش را داشتید به دست نیاورد زیرا نتوانسته اید برنامه خود را راه اندازی کنید. انتشار برنامه در شرکت FSB برای برنامه نویسان یک گزینه بسیار هوشمندانه برای معرفی اپلیکیشن و کسب حداکثر درآمد است.
در شرکت FSB بعید است که اپلیکیشنی برای انتشار پذیرفته نشود. از آنجایی که شرکت FSB نیازی به بررسی برنامه شما ندارد، توسعه دهندگان می توانند سریعتر به جامعه هدف خود برسند.
از آنجایی که شرکت ما بازار بسیار زیادی از فروشندگان و مصرف کنندگان برنامه های دانلودی دارد، بنابراین فرصتی برای توسعه دهندگان وجود دارد که برنامه های خود را به بازار مصرف بزرگ معرفی کنند و از فروش آن کسب درآمد کنند.
به این ترتیب ما برنامه دانلودی شما را برای فروش در وب سایت خود قرار می دهیم و درصدی از فروش اپلیکیشن را به شما پرداخت می کنیم. ابتدا باید در وب سایت ما ثبت نام کنید. سپس درخواست خود را در قسمت بلیط برای ما ارسال کنید. سپس تیم پشتیبانی با شما تماس می گیرد و جزئیات توافق را به شما می گوید.
توضیحات یکی از اعضا :
کمپانی FSB چیست؟
کمپانی FSB ، یک کمپانی آمریکایی_هلندی است که سرور سایت آن در این دو کشور است و از طریق سایت(whoisdomain) قابل رویت است و به مدت ۴ سال است در حوزه‌های مختلف خدمات دیجیتال فعالیت می‌کند.
و اما موضوعی که قراره باهم در اینجا راجبش صحبت کنیم، جدیدترین پروژه این کمپانی است که به خاطر شروع این پروژه دفتر خود را در شهر دبی راه‌اندازی کرده. 
سرمایه گذاری(investing)
این کمپانی استارت پروژه‌ای را در حوزه املاک و مستغلات دبی در متاورس زده و از این‌رو برای جذب کاربر بیشتر و رشد و ارتقا پروژه خود ،شروع به جذب سرمایه گذار کرده و برای جذب کاربر بیشتر طرح‌های درآمدی فوق‌العاده‌ای را در اختیار کاربران خود گذاشته.
طرح‌های در آمدی کمپانی:
در ابتدا لازم به ذکر است هر فرد می‌تواند در حساب کاربری خود از حداقل ۱۰۰تتر(دلار دیجیتال) تا حداکثر 100.000تتر سرمایه گذاری کند(طی قراردادهای۶ماهه،۱۲ماهه،۱۸ماهه،۲۴ماهه که هیچ تفاوتی جز زمان باهم ندارند)
و شما میتوانید تا ۲۳ برابر سرمایه خود را از طریق طرح‌های درآمدی کمپانی کسب نمایید. 
●طرح۱(سود ثابت) :
در این طرح به هر فرد در ازای سرمایه‌گذاری ماهیانه ۱۰٪ سود ثابت تعلق می‌گیرد.💵 که پس از پایان قرار داد اصل پول را نیز می‌توانید بردارید و قابلیت فسخ قرارداد و بازگشت وجه(revok)موجود است
*نکته:در صورت فسخ قرارداد ۳۰٪کارمزد و سودهایی که تا آن لحظه دریافت کرده‌اید کسر میشود 
●طرح۲(باینری) :
در ازای معرفی افراد با کد معرف خود میتوانید به صورت تعادلی ۱۰٪ بالانس خود را پورسانت بگیرید که تسویه آن روزانه است(شرط لازم تعادل دو طرف اکانت شماست).
این طرح درآمدی قویترین طرح ریفرال در دنیاست چون در عمق و بدون محدودیت سطح قابلیت پورسانت دهی دارد. 
●طرح۳(سود از سود):
در این طرح شما میتوانید بدون شرط ۲٪ سود ماهیانه اعضای مجموعه خود تا سطح ۱۵ را به عنوان پاداش ثابت ماهیانه دریافت کنید(برای مثال از سطح۱تا۱۵ شما اگر ۱۰۰هزار دلار سرمایه گذاری شود ماهیانه ۱۰هزار دلار سود حاصل می‌کند که ۲ درصد آن نیز که۲۰۰دلار می‌شود به شما تعلق میگیرد)

بررسی های انجام شده :
طبق توضیحات یکی از اعضا 4 سال از در حال فعالیت است که در اطلاعات whois ، سایت از سال 2020 شروع به فعالیت کرده است,

![3](https://user-images.githubusercontent.com/103103225/183713273-3e8df782-d3ff-4338-a8f8-e3198621e7f4.jpg)

طرح های سرمایه گذاری و ثبت نام و سود دهی مشروط به شبکه سازی است که اصول و پایه شرکت های پانزی است .
بررسی در اسکم ادوایز : 
ترجمه ماشینی : 
هویت صاحب وب سایت در WHOIS پنهان است
رتبه ترانکو (چقدر ترافیک) نسبتاً پایین است
مشاغل PTC ارائه شده در این سایت ممکن است پرخطر باشند

![1](https://user-images.githubusercontent.com/103103225/183713325-4f411aaa-6f59-44ad-971c-e51fb8b732a3.jpg)
![2](https://user-images.githubusercontent.com/103103225/183713333-9fff9706-e6e2-45b3-bb49-f2213a917512.jpg)
![4](https://user-images.githubusercontent.com/103103225/183713340-a2250587-2d5f-41d8-b22a-88cf700424df.jpg)

همچنان اطلاعات افراد گرداننده سایت و پروژه کاملا مخفی است که این نکته خیلی مهم است .
در ایتدای سایت خبر تاسیس دفتر دبی داده شده با یکسری تصاویر ولی هیچ گونه ادرس و تلفنی مبتی بر وجود دفتر داده نشده و صرفا به چند عکس و توضیحات بسنده کرده است .
هیچ گونه ادرس، راه ارتباطی مشخص،دفترمرکزی ، افراد گرداننده پروژه در هیچ جای سایت داده نشده و کاملا مخفی است شاید هم وجود نداشته باشد.
 برای ثبت نام حتما نیازمند معرف هستید .

![7](https://user-images.githubusercontent.com/103103225/183713372-ac54a7b9-26f1-4c0a-a411-576b7a4aadbb.jpg)

تصویر زیر هم گویای خیلی از مطالب است .

![8](https://user-images.githubusercontent.com/103103225/183713425-93faea7f-fb54-40e2-b996-bfb3a70c4a7c.jpg)
![10](https://user-images.githubusercontent.com/103103225/183713443-89072307-05cc-45bc-8cd4-96f43d7c1164.jpg)

برای ثبت نام و ورود به شرکت هم نیاز به پرداخت اولیه دارید که به تترباید پرداخت شود.

![9](https://user-images.githubusercontent.com/103103225/183713487-82633c7a-13e8-45e4-813a-efc553c1a66c.jpg)

ادرس ولت تتر این مجموعه که خودشون جیمیل میکنند جهت واریزی مبلغ ثبت نام  TLgnM1XENt6gLKVrPmxAxssMVz2LSW8hHM

![11](https://user-images.githubusercontent.com/103103225/183713514-10e98003-1747-471d-b2e3-354b82f13c26.jpg)

ادرس دریافت کننده مبالغ ثبت نام تمامی واریزی های خود را به ادرس TDXnWW3Dj3cLEw6hvHprR2J1ZWS326r9G4  واریز میکند و از این ادرس مبالغ در ولت های متفاوتی پخش میشود.

با توجه به اطلاعات بالا این سایت دارای هشدار پانزی و کلاهبرداری است و بر اصل شبکه سازی و ورود افراد جدید فعالیت میکند و برای ثبت نام باید مبلغی وجه واریز کرد و با توجه به این که این جا بر اساس خرید جایگاه ووورد افراد جدید فعالیت میکنند و حتی محصولی موجود نیست به شدت ریسکی است . از واریز وجه و ثبت نام و دادن اطلاعات به چنین کمپانی هایی به شدت خودداری کنید همچنین فعالیت های توضیح داده شده توسط سایت خود شرکت واهی است و واقعیت ندارد.

این بررسی در صورت یافتن اطلاعات جدید به روز خواهد شد .

**Comment by iranhoshdar** [comment]
مطالب تکمیلی در ارتباط با این سیستم کلاهبردار در چنل تلگرام زیر :

https://t.me/FutureSuccessBusiness
